{"name": "test_login_with_invalid_credentials", "status": "passed", "steps": [{"name": "Вход в систему с недействительными учетными данными", "status": "passed", "steps": [{"name": "Регистрация ответа", "status": "passed", "attachments": [{"name": "Текст ответа", "source": "f6c49b09-e704-4611-95be-c83b534d84bf-attachment.json", "type": "application/json"}, {"name": "Заголовки", "source": "d73c074b-df04-4cdc-bf42-9457d743e627-attachment.json", "type": "application/json"}], "start": 1719739227894, "stop": 1719739227894}], "start": 1719739226617, "stop": 1719739227895}], "start": 1719739226617, "stop": 1719739227895, "uuid": "aeba68e5-957e-4ab9-ab87-03da98afa3db", "historyId": "15d5d744ddd2cc4bbf96e98f1487c81f", "testCaseId": "15d5d744ddd2cc4bbf96e98f1487c81f", "fullName": "tests.test_user_operations.TestUserOperations#test_login_with_invalid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('user_page', 'user_data', 'created_user')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_operations"}, {"name": "subSuite", "value": "TestUserOperations"}, {"name": "host", "value": "sanya"}, {"name": "thread", "value": "17088-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_operations"}]}